tox.ini

```diff
@@ -7,8 +7,7 @@ whitelist_externals =
 deps =
     -r{toxinidir}/requirements/test-requirements.txt
 commands =
-    coverage run -m nose {posargs}
-    coverage xml
+    python -m pytest --cov --cov-report=xml {posargs}
```

Comment:
Please note that this is not running the tests against the created package by `tox`, but only the checked-out code (this is true for the previous configuration using `nose`). This might hide issues with the package creation, for example if your package is broken this will succeed as it is using the code and tests from `./diff_cover`, not from the package created by `tox` and installed in the virtual environment.

Reply:
Here's an article which explains the issue quite well: 

https://hynek.me/articles/testing-packaging/

Reply:
Interesting! Ill open it as an issue